src/packages/client/src/generation/generateClient.ts

```diff
@@ -76,7 +76,7 @@ export async function buildClient({
   activeProvider,
 }: GenerateClientOptions): Promise<BuildClientResult> {
   const document = getPrismaClientDMMF(dmmf)
-
+  const useNapi = generator?.previewFeatures.includes('napi')
```

Comment:
```suggestion
  const useNapi = generator?.previewFeatures?.includes('napi')
```

Since it could be undefined it could error with `Uncaught TypeError: can't access property "includes", previewFeatures is undefined`